tools/webpack/blocks.js

```diff
@@ -74,134 +74,209 @@ const createEntrypoints = () => {
+								// block will still call the core function when updates are back ported.
+								content = content.replace(
+									new RegExp(
+										prefixFunctions.join( '|' ),
+										'g'
+									),
+									( match ) =>
+										`${ prefix }${ match.replace(
+											/^wp_/,
+											''
+										) }`
+								);
+
+								// Within content, search for any function definitions. For
+								// each, replace every other reference to it in the file.
+								return (
+									Array.from(
+										content.matchAll(
+											/^\s*function ([^\(]+)/gm
+										)
 									)
-							);
+										.reduce(
+											( result, [ , functionName ] ) => {
+												// Prepend the Gutenberg prefix, substituting any
+												// other core prefix (e.g. "wp_").
+												return result.replace(
+													new RegExp(
+														functionName,
+														'g'
+													),
+													( match ) =>
+														prefix +
+														match.replace(
+															/^wp_/,
+															''
+														)
+												);
+											},
+											content
+										)
+										// The core blocks override procedure takes place in
+										// the init action default priority to ensure that core
+										// blocks would have been registered already. Since the
+										// blocks implementations occur at the default priority
+										// and due to WordPress hooks behavior not considering
+										// mutations to the same priority during another's
+										// callback, the Gutenberg build blocks are modified
+										// to occur at a later priority.
+										.replace(
+											/(add_action\(\s*'init',\s*'gutenberg_register_block_[^']+'(?!,))/,
+											'$1, 20'
+										)
+								);
+							},
+							noErrorOnMissing: true,
 						},
-						noErrorOnMissing: true,
-					},
-					{
-						from: `${ from }/*/block.json`,
-						to( { absoluteFilename } ) {
-							const [ , dirname ] = absoluteFilename.match(
-								new RegExp(
-									`([\\w-]+)${ escapeRegExp(
-										sep
-									) }block\\.json$`
-								)
-							);
+						{
+							from: `${ from }/*/block.json`,
+							to( { absoluteFilename } ) {
+								const [ , dirname ] = absoluteFilename.match(
+									new RegExp(
+										`([\\w-]+)${ escapeRegExp(
+											sep
+										) }block\\.json$`
+									)
+								);
 
-							return join( to, dirname, 'block.json' );
+								return join( to, dirname, 'block.json' );
+							},
 						},
+					] )
+				),
+			} ),
+		].filter( Boolean ),
+	},
+	{
+		entry: {
+			// blockname: './packages/block-library/src/blockname/interactivity.js',
+		},
+		output: {
+			devtoolNamespace: 'wp',
+			filename: './build/block-library/interactive-blocks/[name].min.js',
+			path: join( __dirname, '..', '..' ),
+		},
+		optimization: {
+			runtimeChunk: {
+				name: 'vendors',
+			},
+			splitChunks: {
+				cacheGroups: {
+					vendors: {
+						name: 'vendors',
```

Comment:
That's a good question. 🤔 I guess we can modify the vendors' bundle's definition so it only includes the common libraries, i.e., `preact`, `@preact/signals`, etc., instead of all dependencies from `node_modules`.